ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -473,6 +473,7 @@ let main backend =
     )
     () ;
   Xenops_server.WorkerPool.start !worker_pool_size ;
+  Tracing.Export.set_service_name "xenopsd" ;
```

Comment:
This seems like it should be set up earlier, before xenops starts listening, maybe at around line 433?